etc/kubernetes/manifests/overlays/local/keycloak/data/create-user.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# wait for keycloak to deploy
+kubectl wait --for=condition=Available deployment/cos-fleet-manager-kc
+KC_BASE_PATH=https://$(kubectl get route cos-fleet-manager-kc -o jsonpath={.spec.host})
```

Comment:
as this script is executed inside the cluster, do we need the route ? I guess the service would be enough.
eventually the path of the service  can be provided as an argument of the script

Reply:
We are also setting the frontend URL of the realms to the route on lines 34-37, to allow login using the rhoc tool